### ISSUE TITLE
Missing returned fields in https://ecotaxa.obs-vlfr.fr/api/project_set/user_stats

### ISSUE THREAD

**Issue by grololo06** [ecotaxa/ecotaxa_dev#672]
Specs from #382 : Maybe missing from this entry point are information:
`List of all users (including past ones who might not be associated to the project anymore and whose validations may have been overwritten = in the history table) with the name (link to email), number of annotations (i.e. validation, or set as dubious) and date of last annotation.`

**Comment by grololo06** [comment]
Future extended reply to the query:
```
[
  {
    "projid": 0,
    "annotators": [
      {
        "id": 0,
        "name": "string"
      }
    ],
    "activities": [
      {
        "id": 0,
        "nb_actions": 0,
        "last_annot": "string"
      }
    ]
  }
]
```
The last_annot being Date, with format YYYY-MM-DDThh:mm:ss

**Comment by grololo06** [comment]
Fixed in 2.5.8.
